doc/api/deprecations.md

```diff
@@ -2230,6 +2230,12 @@ like `dns.lookup(false)` due to backward compatibility.
 This behavior is undocumented and is thought to be unused in real world apps.
 It will become an error in future versions of Node.js.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: process.binding('uv').errname() private api
+
+Type: Documentation-only (supports [`--pending-deprecation`][])
+
+
```

Comment:
Possible nit for a lander: extra empty line.

Reply:
@vsemozhetbyt I’d count that under @jasnell’s comment. We want a description here rather than removing empty lines :)